cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -1229,4 +1282,3 @@ readAlonzoGenesis fpath = do
   lbs <- handleIOExceptT (ShelleyGenesisCmdGenesisFileError . FileIOError fpath) $ LBS.readFile fpath
   firstExceptT (ShelleyGenesisCmdAesonDecodeError fpath . Text.pack)
     . hoistEither $ Aeson.eitherDecode' lbs
-
```

Comment:
Is your editor deleting newlines at the end of the file?

Reply:
I don't think so.